content/devdocs/references/p2p_networking.md

```diff
@@ -420,7 +626,16 @@ message header has been omitted and the actual IP addresses have been
 replaced with [RFC5737][] reserved IP addresses.)
 
 {{ <highlight text> }}
-TODO : INTR message hex dump
+IntroductionMessage:
+```
+0e 00 00 00 00 00 00 00 d2 04 00 00 d2 1e 05 00
+00 00 ............................................. Full message
+------------------------------------------------------------------------
+0x0000 0e 00 00 00 00 00 00 00 ........................... Prefix
```

Comment:
This is not quite like it should be . First four bytes are the total size of the buffer that follows . The other four bytes are the message `Prefix` ID (e.g.one of `INTR`, `GETB`, ...) . Please fix that